deployment/ccip/changeset/cs_home_chain.go

```diff
@@ -152,10 +153,9 @@ func deployHomeChain(
 			}
 		})
 	if err != nil {
-		lggr.Errorw("Failed to deploy CCIPHome", "err", err)
+		lggr.Errorw("Failed to deploy CCIPHome", "chain", chain.Name, "err", err)
 		return nil, err
 	}
-	lggr.Infow("deployed CCIPHome", "addr", ccipHome.Address)
```

Comment:
I think we want all the positive logs too

Reply:
According to @AnieeG they were moved to `DeployContract` func